.yarnrc

```diff
@@ -1,3 +1,3 @@
---install.modules-folder "./public/lib"
+--*.modules-folder "./public/lib"
```

Comment:
Whoa, with this as the only change, i really did not expect so many errors -- all are:

```

ERROR LoginFlowTest#test_should_redirect_to_dashboard_when_logging_in_from_/login (122.32s)
Minitest::UnexpectedError:         ActionView::Template::Error: Webpacker can't find application.js in /home/runner/work/plots2/plots2/public/packs-test/manifest.json. Possible causes:
        1. You want to set webpacker.yml value of compile to true for your environment
           unless you are using the `webpack -w` or the webpack-dev-server.
        2. webpack has not yet re-run to reflect updates.
        3. You have misconfigured Webpacker's config/webpacker.yml file.
        4. Your webpack configuration is not creating a manifest.
        Your manifest contains:
        {
        }
        
            app/views/layouts/application.html.erb:110:in `_app_views_layouts_application_html_erb__1573004097732573045_46922334938480'
            test/integration/login_flow_test.rb:68:in `block in <class:LoginFlowTest>'
```

Reply:
The goal here was to respect the modules folder for /all/ yarn commands, not just install!

Reply:
```suggestion
--modules-folder "./public/lib"
```